compiler-plugin/src/main/kotlin/arrow/meta/internal/kastree/ast/psi/Converter.kt

```diff
@@ -937,6 +942,9 @@ internal val PsiElement.ast: Node get() = when(this) {
   is KtValueArgument -> Converter.convertValueArg(this)
   is KtTypeReference -> Converter.convertTypeRef(this)
   is KtClassBody -> Converter.convertClassBody(this)
+  is KtProperty -> Converter.convertDecl(this)
+  is KtPropertyAccessor -> Converter.convertAccessor(this)
```

Comment:
I think we can change this call to `Converter.convertDecl(this)` and add the pattern matching check there (`is KtPropertyAccessor -> convertPropertyAccessor(v)`) to avoid this new function creation since `Property` node is a `Decl` too.

Reply:
thanks for your feedback! this is great :)